code/datums/datumvars.dm

```diff
@@ -1,18 +1,11 @@
-/datum
-	var/var_edited = FALSE //Warrenty void if seal is broken
-	var/fingerprintslast = null
-
 /datum/proc/can_vv_get(var_name)
 	return TRUE
 
 /datum/proc/vv_edit_var(var_name, var_value) //called whenever a var is edited
-	switch(var_name)
-		if ("vars")
-			return FALSE
-		if ("var_edited")
-			return FALSE
-	var_edited = TRUE
+	if(var_name == NAMEOF(src, vars) || (var_name == NAMEOF(src, datum_flags) && (!isnum(var_value) || (!(var_value & DF_VAR_EDITED) && (datum_flags & DF_VAR_EDITED)))))

```

Comment:
There's no problem on letting datum_flags be altered, as line 8 will add the var edited flag later.

Reply:
You could change it to some invalid value but i agree this is probably unneeded and will just complicate things in the future.